easybuild/easyblocks/t/trinity.py

```diff
@@ -294,7 +294,7 @@ def install_step(self):
     def sanity_check_step(self):
         """Custom sanity check for Trinity."""
 
-        path = 'trinityrnaseq_r%s' % self.version
+        path = os.path.basename(self.cfg['start_dir'].strip('/'))
```

Comment:
seems like you stole this from `make_module_req_guess`?

avoid code duplication, introduce a `self.path`, preferably with a more sensible name (and initialise it to `None` first in `__init__`)